frontend/workflows/experimentation/src/list-experiments.jsx

```diff
@@ -1,5 +1,38 @@
 import React, { useState } from "react";
-import { client, Row, Table } from "@clutch-sh/core";
+import { ButtonGroup, client, Row, Table } from "@clutch-sh/core";
+import { Container } from "@material-ui/core";
+
+import { StartAbortExperiment, StartLatencyExperiment } from "./start-experiment";
+
+function renderAbortData(experiment) {
+  const ts = experiment.testSpecification;
+  return (
+    <Row
+      key={experiment.id}
+      data={[
+        ts.abort.clusterPair.downstreamCluster,
+        ts.abort.clusterPair.upstreamCluster,
+        ts.abort.percent,
+        ts.abort.httpStatus,
+      ]}
+    />
+  );
+}
+
+function renderLatencyData(experiment) {
+  const ts = experiment.testSpecification;
+  return (
+    <Row
+      key={experiment.id}
+      data={[
+        ts.latency.clusterPair.downstreamCluster,
+        ts.latency.clusterPair.upstreamCluster,
+        ts.latency.percent,
+        ts.latency.httpStatus,
+      ]}
+    />
+  );
+}
```

Comment:
```suggestion
const renderSpecificationData = (id, data) => (
  <Row
    key={id}
    data={[
      data.clusterPair.downstreamCluster,
      data.clusterPair.upstreamCluster,
      data.percent,
      data.httpStatus,
    ]}
  />
);
```

Reply:
there was actually a bug in the initial code :( The second set of data should include the durationMS instead of httpStatus